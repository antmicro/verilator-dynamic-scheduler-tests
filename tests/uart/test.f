-Wno-WIDTH $TEST_DIR/test.sv $TEST_DIR/verilog-uart/rtl/uart.v $TEST_DIR/verilog-uart/rtl/uart_rx.v $TEST_DIR/verilog-uart/rtl/uart_tx.v $REPO_DIR/src/main_new.cpp
