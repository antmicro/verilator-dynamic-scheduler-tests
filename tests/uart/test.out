Rst end
UART_TX sent: 68                  50
UART_TX sent: 65                  52
UART_TX received: 00                 198
UART_RX received: 68                 206
UART_RX sent: 68                 210
UART_TX sent: 6c                 218
UART_TX received: 68                 360
UART_RX received: 65                 366
UART_RX sent: 65                 370
UART_TX sent: 6c                 380
UART_TX received: 65                 522
UART_RX received: 6c                 526
UART_RX sent: 6c                 530
UART_TX sent: 6f                 542
UART_TX received: 6c                 684
UART_RX received: 6c                 686
UART_RX sent: 6c                 690
UART_TX sent: 20                 704
UART_TX received: 6f                 846
UART_RX received: 6f                 846
UART_RX sent: 6f                 850
UART_TX sent: 77                 866
UART_RX received: 20                1006
UART_TX received: 20                1008
UART_RX sent: 20                1010
UART_TX sent: 6f                1028
UART_RX received: 77                1166
UART_TX received: 77                1170
UART_RX sent: 77                1170
UART_TX sent: 72                1190
UART_RX received: 6f                1326
UART_RX sent: 6f                1330
UART_TX received: 6f                1332
UART_TX sent: 6c                1352
UART_RX received: 72                1486
UART_RX sent: 72                1490
UART_TX received: 72                1494
UART_TX sent: 64                1514
UART_RX received: 6c                1646
UART_RX sent: 6c                1650
UART_TX received: 6c                1656
UART_RX received: 64                1806
UART_RX sent: 64                1810
Data received successfully
UART_TX received: 64                1818
Data sent successfully
Finished!
